app/src/main/java/vn/edu/usth/minigh/closedtabFragment.java

```diff
@@ -0,0 +1,35 @@
+package vn.edu.usth.minigh;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class closedtabFragment extends Fragment {
+
+    public closedtabFragment() {
```

Comment:
```suggestion
public class ClosedTabFragment extends Fragment {

    public ClosedTabFragment() {
```

Also I don't think we need any of the methods below.

Reply:
ok I'll change all the classes name in PascalCase then push.